### PR TITLE
Adding a command

### PR DESCRIPTION
**Pull request by Askirkela**
* Adding a `!sam` filter that prints something like "Sa mère la vertueuse" or "Sa génitrice la prostipéripatétipute"
* Adding `addmom` command to add something to the mom array
* Adding `addyomom` command to add something to the insult array

> The first PR is always messy, deal with it!
> Askirkela, 2017

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -10,96 +10,116 @@ rtm.start();
...
+        }
     }
-  }
 }
 
 let applyFilter = true;
 const store = {}
 
+const mom = ["Sa mère", "Sa maman", "Sa génitrice"];
+const yomom = ["la prude", "la vertueuse", "la prostipéripatétipute"];
```
Comment (fteychene) [inline main.js]: maybe rename `yomon` to `momAdjective` could be more explicit.

**Review by fteychene — CHANGES_REQUESTED** [review]
Just renaming `yomom` and it'll be ok to merge.

**Comment by Askirkela** [comment]
Done

**Review by fteychene — APPROVED** [review]
LGTM

**Pull request merged**